wp-admin/includes/class-wp-plugin-dependencies.php

```diff
@@ -363,12 +363,16 @@ public function modify_plugin_row_elements( $plugin_file, $plugin_data ) {
 	 */
 	public function modify_plugin_row_elements_requires( $plugin_file ) {
 		$names = $this->get_requires_plugins_names( $plugin_file );
-		if ( ! empty( $names ) ) {
-			$names = $this->get_view_details_link( $plugin_file, $names );
-			print '<script>';
-			print 'jQuery("tr[data-plugin=\'' . esc_attr( $plugin_file ) . '\'] .plugin-version-author-uri").append("<br><br><strong>' . esc_html__( 'Requires:' ) . '</strong> ' . wp_kses_post( $names ) . '");';
-			print '</script>';
+
+		if ( empty( $names ) ) {
+			return;
 		}
+
+		$links = $this->get_view_details_link( $plugin_file, $names );
```

Comment:
Shouldn't this be `$this->get_view_details_links`?

Reply:
This is handled in #25 which renames the method.

Reply:
@afragen I've resolved the conflict caused by you being a problem in my life ❤️